firestore-bigquery-export/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.6
+
+fixed - Record document id in schema view and revert timestamp change (#357)
+feature - Record document id of changes tracked by firestore-bigquery-change-tracker package (#374)
```

Comment:
feature - Record document ID of changes tracked by firestore-bigquery-change-tracker package (#374)

Reply:
Done.